tuner/Mono.Tuner/RemoveResources.cs

```diff
@@ -33,19 +33,27 @@ public virtual void Process (LinkContext context)
 				if (resource == null)
 					continue;
 
-				switch (resource.Name) {
+				if (!SkipResource (resource.Name)) {
```

Comment:
The logic seems to be inverse you skip resource on `!SkipResource` ?

Reply:
You are right, the name is wrong. I had inverted it and did not go back to fix it.

Good catch.